assets/java-spring/README.md

```diff
@@ -0,0 +1,19 @@
+# Spring Boot Trivial App
+
+This contains the built JAR, and the source code for a trivial SpringBoot app.
+
+The app has one feature which is to return `ok`.
```

Comment:
We opted to leave this behavior so that no tests required changes.

It would probably be helpful to future CATs operators / maintainers if this app returned something more meaningful, like `Hello from 'assets/java-spring/'` or something. Perhaps....

Reply:
See [[assets/java-spring/src/main/java/org/cloudfoundry/SpringBootTrivialApplication.java](https://github.com/cloudfoundry/cf-acceptance-tests/pull/694/files#diff-dee21702aae4a8ddb642ef9fbbf2cf8aabde12857406888312c4de2028b4fd17)](https://github.com/cloudfoundry/cf-acceptance-tests/pull/694/files#r1063008729) below